firmware-management/swupdate-handlers/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(swupdate-handlers)
+
+add_library(swupdate-handlers SHARED rootfs-handler.c arm-handler-common.c)
+
+target_include_directories(swupdate-handlers PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${CMAKE_CURRENT_SOURCE_DIR}/include/swupdate ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
> `${CMAKE_CURRENT_SOURCE_DIR}/include/swupdate`

This will be removed once we start using the `swupdate-headers` recipe to get the headers, right?

Reply:
Actually, with the code in its current state, I don't think we need it because all swupdate `#include`s already start with the `swupdate/` string.

Reply:
I had already changed this locally to 
```
target_include_directories(swupdate-handlers PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
```

Reply:
Okay, cool :-)